### PR TITLE
Evenementen

### PR DESCRIPTION
**Pull request by HuubHamstra**
Made search bar, layout and tables in evenementen.php

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -170,4 +169,124 @@ footer {
...
\ No newline at end of file
+}
+
+#eventstitle{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    text-align: center;
+    font-size:4em;
```
Comment (jariolyslager) [inline css/style.css]: no space after :

File: css/style.css [hunk 2]
```diff
@@ -170,4 +169,124 @@ footer {
...
+    grid-column: 1/5;
+    background-color: white;
+}
+
+#purplebarwidth{
+	grid-column: 1/5;
```
Comment (jariolyslager) [inline css/style.css:188]: 1 tab too many

File: css/style.css [hunk 3]
```diff
@@ -170,4 +169,124 @@ footer {
...
+    border-right: 1px solid white;
+    border-left: 1px solid white;
+    padding: 0.625em;
+    text-align: left;
+}
+
+#addcontainer, #linkcontainer {
+    align-items: center;
+    text-align: center;
+    font-size:1em;
```
Comment (jariolyslager) [inline css/style.css]: no space after :

File: eventadd.php [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    $err = [];
+    if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+        $statement = $dbHandler->prepare("INSERT INTO Event (CustomerID, EventName, Location, Price, Date, Description) 
+                                                VALUES (:CustomerID, :EventName, :Location, :Price, :Date, :Description);");
+
+        if (!$CustomerID = filter_input(INPUT_POST, 'CustomerID', FILTER_VALIDATE_INT)) {
+            $err[] = "Vergeten het klant ID toe te voegen";
+        }
+        if (!$EventName = filter_input(INPUT_POST, 'EventName', FILTER_SANITIZE_SPECIAL_CHARS)) {
+            $err[] = "";
```
Comment (jariolyslager) [inline eventadd.php]: empty error messages

**Review by jariolyslager — COMMENTED** [review]
Added some comments. Most important is no error messages in eventadd.php.

**Review by jariolyslager — APPROVED** [review]

**Review by Kay-Westerhof — APPROVED** [review]

**Review by Jasirabdikarim — APPROVED** [review]

**Pull request merged**